backend/app/app.ts

```diff
@@ -28,7 +28,8 @@ const backend = new serverConstructor({
     bodyParser.json(),
     bodyParser.urlencoded({ extended: true }),
     cors({
-      origin: [FRONTEND_URL, 'http://localhost:8080', 'http://localhost:3000', 'http://localhost'],
+      origin: [FRONTEND_URL, 'http://localhost:8080', 'http://localhost:3000', 'http://localhost',
+        'http://seaqull-test.com:8080'],
```

Comment:
```suggestion
      origin: [
        FRONTEND_URL,
        'http://localhost:8080',
        'http://localhost:3000',
        'http://localhost',
        'http://seaqull-test.com:8080'
      ],
```

This looks a bit better tbh

Reply:
Also, you should change the seaqull-test.com to seaqull or just remove it